README.md

```diff
@@ -66,6 +73,8 @@ The `sns-testing` solution is based on Docker; however, there are subtle issues
    mv "${DFX_NET_JSON}.tmp" "$DFX_NET_JSON" 2>/dev/null  # restore original config if it was present
    ```
 
+   You might have to hit ``Allow'' on the Mac popup which happens when you run `./bin/dfx` for the first time.
```

Comment:
I think this happened when I ran either basic_scenario.sh or the step before, but I forget offhand.

Reply:
Are you suggesting to drop this note if you're not sure about the error anymore?

Reply:
```suggestion
   While running these instructions for the first time, you may need to hit the ``Allow'' button to authorize the system to execute the binaries shipped with sns-testing, e.g., `./bin/dfx`.
```